app/components/predicaat-input.js

```diff
@@ -0,0 +1,57 @@
+import Component from '@glimmer/component';
+
+import { tracked } from '@glimmer/tracking';
+
+import {
+  API,
+  INPUT_DEBOUNCE,
+  JSON_API_TYPE,
+} from 'frontend-lmb/utils/constants';
+
+import {} from 'reactiveweb/function';
+import { task, timeout } from 'ember-concurrency';
+
+export default class PredicaatInput extends Component {
+  @tracked uri;
+  @tracked errorMessage;
+
+  get predicaatUri() {
+    return this.uri || this.args.value;
+  }
+
+  onUpdate = task({ restartable: true }, async (event) => {
```

Comment:
this needs to be a droptask i think, right now it deletes part of my string if i want to type a long uri